packages/squiggle-lang/src/rescript/ReducerInterface/ReducerInterface_ExpressionValue.res

```diff
@@ -5,26 +5,33 @@
 module Extra_Array = Reducer_Extra_Array
 module ErrorValue = Reducer_ErrorValue
 
+@genType.opaque
+type internalCode = Object
```

Comment:
I'm not sure I understand the semantics, but surely we can do better at naming this instead of `Object`.

Reply:
Object (Reducer Standard Library) corresponds to TypeScript Any.

Reply:
I'm a tad confused as to why you are using Object rather than expression? Is there a story behind that?